### PR TITLE
ASCII plugin

### PR DESCRIPTION
**Pull request by clementroche**
This plugin allows users to apply an ascii filter to assets. It accepts images, videos and gltf models.

### CODE REVIEW

File: plugins/ascii/package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        "eslint-plugin-react-hooks": "^4",
+        "eslint-plugin-react-refresh": "^0",
+        "prettier": "^3.2.5",
+        "prettier-plugin-organize-imports": "^3.2.4",
+        "typescript": "^5.4.5",
+        "vite": "^5.2.9",
+        "vite-plugin-framer": "^0.0.9"
+    },
+    "optionalDependencies": {
+        "@rollup/rollup-linux-x64-gnu": "4.9.5"
```
Comment (huntercaron) [inline plugins/ascii/package.json:45]: Is this being used anywhere?

File: plugins/ascii/public/suzanne.glb
Comment (huntercaron) [inline plugins/ascii/public/suzanne.glb:1]: Is this being used?

File: plugins/ascii/public/default.png
Comment (huntercaron) [inline plugins/ascii/public/default.png:1]: Is this being used?

File: plugins/ascii/public/ascii.svg
Comment (huntercaron) [inline plugins/ascii/public/ascii.svg:1]: We can remove this yeah?

File: plugins/ascii/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+}
+// export function useSelectedNode() {
+//     const [selection, setSelection] = useState<CanvasNode[]>([])
+
+//     useEffect(() => {
+//         return framer.subscribeToSelection(setSelection)
+//     }, [])
+
+//     return selection
+// }
```
Comment (huntercaron) [inline plugins/ascii/src/App.tsx:39]: we can cleanup?

File: plugins/ascii/src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+        // framer.hideUI()
+        // await framer.setImage({
+        //     image: {
+        //         bytes: nextBytes,
+        //         mimeType: originalImage.mimeType,
+        //     },
+        // })
+        // void framer.closePlugin("Image saved...")
+
+        // console.log("total duration", performance.now() - start)
```
Comment (huntercaron) [inline plugins/ascii/src/App.tsx:161]: cleanup?

File: plugins/ascii/src/dropzone/drag-and-drop.tsx [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+//     //         //     } else {
+//     //         //         console.error("Could not generate thumbnail blob.")
+//     //         //     }
+//     //         // }, "image/png")
+//     //     },
+//     //     { once: true }
+//     // )
+
+//     setter({ type: "video", src: url })
+// }
```
Comment (huntercaron) [inline plugins/ascii/src/dropzone/drag-and-drop.tsx:128]: more commented out code

**Review by huntercaron — COMMENTED** [review]
Can you remove the commented out code? Otherwise LGTM

File: plugins/ascii/package.json (hunk 1 above)
Comment (clementroche) [reply plugins/ascii/package.json:45]: it was needed to build the project into vercel

**Pull request closed without merging**